### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by vagarwala** [Viliami/Color-Switch#1]
Hi, I came across this on github and tried to run it on my own, but when I did so I got a segfault.  Am I doing something wrong or...?
![screen shot 2016-06-19 at 11 33 16 am](https://cloud.githubusercontent.com/assets/12286367/16178291/bdb514bc-3611-11e6-9b73-bd556022653e.png)

**Comment by Viliami** [comment]
I made this game using Python 3.2, you are using Python 2.7.

This program also uses pygame, also I've changed some of the code, because I hard coded a directory, please download the updated game.py file.

**Comment by vagarwala** [comment]
Yes I saw the hard coded directory and had already changed that.  Trying with Python 3.2 now.
